### PR TITLE
EOS-19795: Performance - KV Index access optimization

### PR DESCRIPTION
**Pull request by evgeniy-brazhnikov**
Signed-off-by: evgeniy-brazhnikov <evgeniy.brazhnikov@seagate.com>

### CODE REVIEW

File: server/motr_delete_key_value_action.cc [hunk 1]
```diff
@@ -70,7 +70,7 @@ void MotrDeleteKeyValueAction::delete_key_value() {
           request, s3_motr_api);
     }
     motr_kv_writer->delete_keyval(
-        index_id, request->get_key_name(),
+        {index_id}, request->get_key_name(),
```
Comment (knrajnambiar76) [inline server/motr_delete_key_value_action.cc:73]: whats the need of {...}

**Review by knrajnambiar76 — APPROVED** [review]
Please do test it thoroughly.

File: server/motr_delete_key_value_action.cc (hunk 1 above)
Comment (evgeniy-brazhnikov) [reply server/motr_delete_key_value_action.cc:73]: It's because the signature of delete_key_value() was changed

**Comment by t7ko-seagate** [comment]
Closing this PR as #951 is the follow-up, #951 includes all changes from this PR.  This PR was "work in progress", raised for early review.

**Pull request closed without merging**